src/deltaSync/types.ts

```diff
@@ -2,12 +2,16 @@ import { User } from '../users/types';
 import { Task } from '../tasks/types';
 import { Project } from '../projects/types';
 import { TimeTracking } from '../timetrackings/types';
+import { AbsenceType } from '..';
+import { GeneralSetting } from '../generalSettings/types';
 
 export type SyncData = {
   users: User[];
   tasks: Task[];
   projects: Project[];
   timeTrackings: TimeTracking[];
+  absenceTypes: AbsenceType[];
```

Comment:
We need to refactor the SyncData, because in the end we will have all endpoint here...

Reply:
Not all endpoints are supported in backend of deltasync.